src/titiler_fetcher.mjs

```diff
@@ -1,9 +1,15 @@
 import got from "got";
 import PQueue from "p-queue";
+import os from "node:os";
+
+// Получаем количество ядер
```

Comment:
@coderabbitai remember to remind to write comments in english and suggest translations

Reply:
Fixed